osu.Game.Tests/Visual/Online/TestSceneCommentsContainer.cs

```diff
@@ -37,23 +39,29 @@ public class TestSceneCommentsContainer : OsuTestScene
         public TestSceneCommentsContainer()
         {
             BasicScrollContainer scroll;
-            CommentsContainer comments;
+            TestCommentsContainer comments;
 
             Add(scroll = new BasicScrollContainer
             {
                 RelativeSizeAxes = Axes.Both,
-                Child = comments = new CommentsContainer()
+                Child = comments = new TestCommentsContainer()
             });
 
             AddStep("Big Black comments", () => comments.ShowComments(CommentableType.Beatmapset, 41823));
             AddStep("Airman comments", () => comments.ShowComments(CommentableType.Beatmapset, 24313));
             AddStep("Lazer build comments", () => comments.ShowComments(CommentableType.Build, 4772));
             AddStep("News comments", () => comments.ShowComments(CommentableType.NewsPost, 715));
+            AddStep("Trigger user change", comments.User.TriggerChange);
             AddStep("Idle state", () =>
             {
                 scroll.Clear();
-                scroll.Add(comments = new CommentsContainer());
+                scroll.Add(comments = new TestCommentsContainer());
             });
         }
+
+        private class TestCommentsContainer : CommentsContainer
+        {
+            public new Bindable<User> User => base.User;
```

Comment:
no need to expose a bindable bound to a source you can easily retrieve.

Reply:
i'm ok with this for now